### ISSUE TITLE
Error in server console when placing order

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#1595]
I am migrating this from a comment on #1585 

Seeing this error in the server console after placing an order

```
03:08:52.788Z ERROR Reaction: Error in orders/sendNotification for order W2BGyczBMJoqQ9egH
  Error: Access Denied [403]
      at [object Object].ordersRefundsList (server/methods/core/orders.js:857:13)
      at packages/check.js:130:16
      at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
      at Object.exports.Match._failIfArgumentsAreNotAllChecked (packages/check.js:129:41)
      at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1709:18)
      at packages/ddp-server/livedata_server.js:1625:18
      at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
      at Server.apply (packages/ddp-server/livedata_server.js:1624:45)
      at Server.call (packages/ddp-server/livedata_server.js:1567:17)
      at packages/kadira:flow-router-ssr/server/plugins/ssr_data.js:67:23
      at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
      at Object.Meteor.(anonymous function) [as call] (packages/kadira:flow-router-ssr/server/plugins/ssr_data.js:66:44)
      at [object Object].ordersSendNotification (server/methods/core/orders.js:390:33)
      at packages/check.js:130:16
      at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
      at Object.exports.Match._failIfArgumentsAreNotAllChecked (packages/check.js:129:41)
      at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1709:18)
      at packages/ddp-server/livedata_server.js:1625:18
      at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
      at Server.apply (packages/ddp-server/livedata_server.js:1624:45)
      at Server.call (packages/ddp-server/livedata_server.js:1567:17)
      at packages/kadira:flow-router-ssr/server/plugins/ssr_data.js:67:23
```

**Comment by kieckhafer** [comment]
@zenweasel I believe this was fixed in a different PR than #1585, and I cannot replicate. Can you please check again if you are still seeing this in release-0.18.0

**Comment by zenweasel** [comment]
Can't replicate. Closing

**Comment by zenweasel** [comment]
Actually seeing this error still. Reopening. Need to create better steps to replicate.

**Comment by zenweasel** [comment]
On the `release-0.18.0` branch, I can replicate this 100% if I create a non-admin user and place an order I get this error message every time.

**Comment by zenweasel** [comment]
Closed via #1607
